src/pages/Register.js

```diff
@@ -105,11 +159,77 @@ const Register = () => {
                 id,
                 value: email,
                 onChange: (v) => setEmail(v),
-                style: { width: '50ex' },
+                style: { width: '66ex' },
+              }),
+            ]),
+          ]),
+      ]),
+      h(IdContainer, [
+        (id) =>
+          div({ style: { lineHeight: '170%' } }, [
+            label({ htmlFor: id, style: { display: 'block', marginTop: '2rem' } }, ['Organization *']),
+            div([
+              h(TextInput, {
+                id,
+                value: institute,
+                onChange: (v) => setInstitute(v),
+                disabled: !partOfOrganization,
+                style: { width: '66ex' },
```

Comment:
Wow, I don't think I've ever seen "ex"! It's certainly not something we typically use, but I see there was a pre-existing usage in this file.